ops/env.conf

```diff
@@ -1,9 +1,13 @@
 env TAG;
 env DB_ADAPTER;
+env MANAGEMENT_TAG
 env POSTGRES_HOST;
 env POSTGRES_DB;
 env POSTGRES_USER;
 env POSTGRES_PASSWORD;
 env PASSENGER_APP_ENV;
+env REGISTRY_HOST;
```

Comment:
I'm not sure we're going to need the REGISTRY_HOST in this information, since I believe we're exclusively using dockerhub as our registry host. Same goes for above.

Reply:
Removed REGISTRY_HOST here